Signal/src/ViewControllers/GifPicker/GifPickerCell.swift

```diff
@@ -188,6 +212,50 @@ class GifPickerCell: UICollectionViewCell {
         }
         imageView.image = image
         self.backgroundColor = nil
+
+        if self.isCellSelected {
+            let activityIndicator = UIActivityIndicatorView(activityIndicatorStyle: .gray)
+            self.activityIndicator = activityIndicator
+            addSubview(activityIndicator)
+            activityIndicator.autoCenterInSuperview()
+            activityIndicator.startAnimating()
+
+            // Render activityIndicator on a white tile to ensure it's visible on
+            // when overalayed on a variety of potential gifs.
```

Comment:
overlayed

Reply:
CALayer makes it easy to add a drop shadow - I wonder if we should use a subtle one to differentiate the tile from the background?

Reply:
done.